ostelco-core/ModelControllers/UserDefaultsWrapper.swift

```diff
@@ -14,6 +14,7 @@ public struct UserDefaultsWrapper {
     private enum Key: String, CaseIterable {
         case pendingEmail = "PendingEmail"
```

Comment:
I guess we could remove this, if it's only used with our firebase login

Reply:
My plan was to cleanup all unused classes, enums and storyboards in a different PR

Reply:
That makes sense 👌